app/src/main/java/app/revanced/integrations/patches/PlayerTypeHookPatch.java

```diff
@@ -26,6 +28,8 @@ public static void YouTubePlayerOverlaysLayout_updatePlayerTypeHookEX(@Nullable
         final PlayerType newType = PlayerType.safeParseFromString(type.toString());
         if (newType != null) {
             PlayerType.setCurrent(newType);
+            SponsorBlockView.playerTypeChanged(newType);
+            SponsorBlockUtils.playerTypeChanged(newType);
```

Comment:
Instead of calling this, `PlayerType` should be read by `SponsorBlockView` and `SponsorBlockUtils`. The reason is that this class has nothing to do with SponsorBlock and separation of the `sponsorblock` patch later on would cause issues.